frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings.tsx

```diff
@@ -59,26 +65,61 @@ export const AppearanceSettings = ({
   );
   const utmTags = `?utm_source=${plan}&utm_media=static-embed-settings-appearance`;
 
-  const fontControlLabelId = useUniqueId("display-option");
   const downloadDataId = useUniqueId("download-data");
 
   return (
     <>
       <StaticEmbedSetupPaneSettingsContentSection
-        title={t`Customizing your embed’s appearance`}
+        title={t`Customizing look and feel`}
       >
-        <Text>{jt`These cosmetic options requiring changing the server code. You can play around with and preview the options here, and check out the ${(
+        <Text>{jt`These options require changing the server code. You can play around with and preview the options here. Check out the ${(
           <ExternalLink
             key="doc"
             href={`${docsUrl}${utmTags}#customizing-the-appearance-of-static-embeds`}
           >{t`documentation`}</ExternalLink>
         )} for more.`}</Text>
       </StaticEmbedSetupPaneSettingsContentSection>
       <StaticEmbedSetupPaneSettingsContentSection
-        title={t`Playing with appearance options`}
         mt="2rem"
```

Comment:
design-wise, do we still need the margin-top here if there is no title? (this is out-of-scope for this PR, but just a reminder)